modules/dms-endpoint/variables.tf

```diff
@@ -104,6 +104,12 @@ variable "mongodb_settings" {
   default     = null
 }
 
+variable "postgres_settings" {
```

Comment:
Yes! This would in fact be my favored approach and might in fact be required to address the issue of 'completness' above. I used any because I wanted to keep postgres_settings consistent with the other `*_settings` variables.

I will try to get another MR in today using an object instead of any.